idb/postgres/ledger_for_evaluator_test.go

```diff
@@ -0,0 +1,588 @@
+package postgres
```

Comment:
This should either live next to `ledger_for_evaluator.go` or be renamed

Reply:
Agree. But we first need to move the sql schema out of the `postgres` package. Otherwise, `postgres` will depend on `ledger_for_evaluator` and vice-versa.

Reply:
At least add comments pointing at each other. for this and writer.
foo.go should have a comment `// tested in somewhere/else/foo_test.go`
and foo_test.go should have a comment `// tests somewhere/else/foo.go` and describe the dependency mess that has it separated from its foo.go